valentyusb/usbcore/cpu/eptri.py

```diff
@@ -66,41 +114,8 @@ def __init__(self, usb_core):
         self.have_data_stage = Signal()
 
         self.submodules.data = buf = ClockDomainsRenamer({"write": "usb_12", "read": "sys"})(ResetInserter()(fifo.SyncFIFOBuffered(width=8, depth=10)))
-        #w {
```

Comment:
I really like having the documentation for the register be right next to the register it's defining.  Is there any way we can do that, rather than having it spread apart?